eng/scale-set-extension-scripts/readme.md

```diff
@@ -0,0 +1,3 @@
+These two scripts are used to configure a Virtual Machine Scale Set extension that's responsible for setting Application Insights environmental variables, and configuring TLS.
+
+If at some point we need to recreate Maestro, we'll need to upload these scripts to a storage account, and add a Scale Set extension that's calling them with the `AppInsightsConnectionString` parameter set
```

Comment:
Bicep could do this, however you'd need a storage account with these scripts inside of it already. But ideally you'd create the storage account with the bicep I think, and do this afterwards.

At some point, when we migrate Maestro to the Pcs, this will no longer be relevant since we set the env variable during creation there

Reply:
Which storage account are these on right now? Do we know?